1904-11-10.xml

```diff
@@ -0,0 +1,2409 @@
+                        throughout the Venezuelan mess was conspicuously moderate and perspicacious,
+                        and made it clear that while he would resist any attemptat the partition or
+                        absorption of a South American Republic by a European Power he would not
+                        otherwise protect an offending Republic from proper punishment for its
+                        misdeeds. In other words, he construes the Monroe Doctrine as entailing on
+                        its beneficiaries the obligation, as he puts it, of "acting with decency."So
+                        far from being an extension of the doctrine,this is rather a contraction of
+                        it, and a translation of its indefiniteness into terms of reason and
+                        practicality. Europe could wish for no better guarantee that the South
+                        American States will falfil their obligations and observe the ordinary
+                        comities of civilisation than the assurance that the United States will only
+                        side with them in a just and reasonable cause. For the rest the success of
+                        Mr. Roosevelt’s decisive action in Panama is its best defence while his
+                        resoluteness in ordering an American squadron first to Beyrout on the rumour
+                        that an American Vice-Consul had been murdered; secondly, to Tangier when
+                        Mr. Perdicaris was captured by the Moorish brigands; and thirdly, to Smyrna
+                        when Turkish dilatoriness had worn out American patience, will be best
+                        appreciated by those who have studied Turkish and Moorish diplomacy most
+                        closely. Each was a bold stroke; but the boldness, as the issue proved, was
+                        that of knowledge and swift adaptation of means to ends. Mr. Roosevelt, when
+                        called on to justify his foreign policy, need only point to its results. In
+                        his home policy, too, he has been equally successful, for, though the
+                        success of his Anti Trust policy may have alienated Wall Street, the average
+                        citizen applauded it. Taken as a whole as a man, a party leader, and a
+                        President, Mr. Roosevelt well deserves his victory, and the world at large
+                        will congratulate the citizens of the United States on having reelected him
+                        to the Presidency.</p>
+                
+                </div>
+            
+            </div>
+            <pb n="3"/>
+            <div type="page" n="3" facs="https://raw.githubusercontent.com/dig-eg-gaz/page-images/master/YYYY-page-images-N/YYYY-MM-DD-p3.jpg"> 
+        <cols n="4"/>
+           <cb n="1"/>
+               <div type="item">
```

Comment:
the feature="wire" div is complex. Treat cables as individual divs (type="cable"), each with its own <dateline> and <title>. This section is sometimes spread over two pages; when this is the case, you can use the next and prev attributes. Look at a complete file and copy the format.